releasing/create-release.sh

```diff
@@ -65,8 +65,16 @@ function build_kustomize_binary {
       fi
       rm output/kustomize
     done
-    rmdir output
+
+  done
+
+  # create checksums.txt
+  for release in "${release_dir}"/*; do
+    sha256sum "$release" > output/checksums.txt
```

Comment:
```suggestion
    sha256sum "$release" >> output/checksums.txt
```
Append or overwrite?

Reply:
Yes, I missed it.
I already fix that.